docs/tutorials/field_type/7_add_content_and_edit_views.md

```diff
@@ -1,275 +1,116 @@
+``` yml
+// Resources/config/fieldtypes.yml
 
-<div class="ez-fieldview-row pure-g">
-    <div class="ez-fieldview-label pure-u">
-        <p class="ez-fieldview-name"><strong>{{ translate_property fieldDefinition.names }}</strong></p>
-    </div>
-    <div class="ez-fieldview-value pure-u"><div class="ez-fieldview-value-content">
-        {{#if isEmpty }}
-            {{translate 'fieldview.field.empty' 'fieldview'}}
-        {{else}}
-            {{ value.url }}
-        {{/if}}
-    </div></div>
-</div>
+    ezsystems.tweetbundle.fieldtype.eztweet.form_mapper:
+        class: EzSystems\TweetFieldTypeBundle\eZ\Publish\FieldType\Tweet\FormMapper
+        tags:
+            - {name: ez.fieldFormMapper.value, fieldType: eztweet}
+        arguments: ['@ezpublish.api.service.field_type']
 ```
 
-### Javascript view file
+# DataTransformer
 
-Next, we need to implement Javascript file that will be responsible for handling the template:
+As mentioned earlier, we also need to tell our FormMapper how to transform ValueObject into a single string which contains URL to given tweet. We do this by creating an implementation of DataTransformerInterface.
 
-```javascript
-//Resources/public/js/views/fields/ez-tweet-view.js
+``` php
+// Form/TweetValueTransformer.php
+namespace EzSystems\TweetFieldTypeBundle\Form;
 
-YUI.add('ez-tweet-view', function (Y) {
-    "use strict";
-    /**
-     * Provides the Tweet field view
-     *
-     * @module ez-tweet-view
-     */
-    Y.namespace('eZ');
+use Symfony\Component\Form\DataTransformerInterface;
+use eZ\Publish\API\Repository\FieldType;
+use eZ\Publish\Core\FieldType\Value;
 
+class TweetValueTransformer implements DataTransformerInterface
+{
     /**
-     * The Tweet field view
-     *
-     * @namespace eZ
-     * @class TweetView
-     * @constructor
-     * @extends eZ.FieldView
+     * @var FieldType
      */
-    Y.eZ.TweetView = Y.Base.create('tweetView', Y.eZ.FieldView, [], {
-        /**
-         * Returns the value to be used in the template. If the value is not
-         * filled, it returns undefined otherwise an object with a `url` entry.
-         *
-         * @method _getFieldValue
-         * @protected
-         * @return Object
-         */
-        _getFieldValue: function () {
-            var value = this.get('field').fieldValue, res;
+    private $fieldType;
 
-            if ( !value || !value.url ) {
-                return res;
-            }
-            res = {url: value.url};
+    public function __construct(FieldType $fieldType)
+    {
+        $this->fieldType = $fieldType;
+    }
 
-            return res;
+    public function transform($value)
+    {
+        if (!$value instanceof Value) {
```

Comment:
In a tutorial (aka to learn) could we use a strict comparison, please?

Reply:
To be honest, I don't have a single clue what do you mean by strict comparison in case of 'instanceof' operator. Could you explain it to me?

Reply:
Strict comparison is `===` instead of `==`
Here it could have been
`if ( false === $value instanceof Value)`
Maybe it has no impact here.

Reply:
To be honest, I am totally against something like this. instanceof is not a function but an operator. Also afair it is against all styling guides.

Reply:
I agree with @ViniTou, the current version is correct